content/en/docs/installation/authentication.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Authentication"
+weight: 40
+description: >
+  Guide to set up basic authentication with Fission
+---
+
+## Authentication for Fission Functions
+
+When using Fission, if you are using an ingress, you might already have some form of authentication in place for external calls.
+But if you aren't, Fission didn't provide a way to configure authentication for your API calls.
+
+Starting with [v1.16.0](/docs/releases/v1.16.0-rc1/), Fission allows you to have an **authentication mechanism in place for Fission function invocations**.
+
+## Understanding Authentication for Fission Functions
+
+Fission does so by enabling authentication for Fission Router.
+This is an optional feature that can be enabled/disable depending on your requirement.
+
+When enabled, a new endpoint for authentication will be registered in the router.
+All the API calls to Fission functions will now be routed through this endpoint.
```

Comment:
All the API calls to Fission functions will be with authentication token. 

We dont route function calls through single endpoint. Endpoints for functions are same ony, only we ensure they are done with auth.